src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1562,12 +1562,11 @@ public class ClusterVNode<TStreamId> :
 			AddTask(perSubscrQueue.Start());
 			AddTask(redactionQueue.Start());
 
-			if (Runtime.IsUnixOrMac) {
```

Comment:
[`PosixSignalRegistration` on Windows emulates`SIGHUP`](https://github.com/dotnet/runtime/blob/2e8f818677e55ef7570b9cd3b21890fa9ac93b0f/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/PosixSignalRegistration.Windows.cs#L87C1-L89C27) so I removed this check.